### ISSUE TITLE
Control Plane not reaching Ready State on Proxmox

### ISSUE THREAD

**Issue by tylerobara** [siderolabs/talos#8432]
## Bug Report
After running apply config to create control plane the VM reboots and attempts to configure but produces errors and does not reach ready state

### Description
Pre-apply-config:
<img width="1277" alt="Screenshot 2024-03-13 at 14 04 00" src="https://github.com/siderolabs/talos/assets/49762914/c34b0b22-be8a-4b76-8a34-d6a8c628bdc3">

Config commands:
```bash
talosctl gen config talos-proxmox-cluster https://192.168.20.40:6443 --output-dir _out
talosctl apply-config --insecure --nodes 192.168.20.40 --file _out/controlplane.yaml
```

Post apply config & reboot:
<img width="1279" alt="Screenshot 2024-03-13 at 14 06 15" src="https://github.com/siderolabs/talos/assets/49762914/c4059925-d397-42ae-b615-85b4281929a9">

Notes: 
- PVE v8.1.4
- IP is set via static lease in OPNSense
- VM > Proxmox Host > Switch (VLAN20) > Router
- Other VMs on Proxmox host pull DHCP and static leases w/o issue

### Logs
```bash
❯ talosctl support -n 192.168.20.40
Processed with errors:
error constructing client: failed to determine endpoints
Support bundle is written to support.zip
❯ talosctl support -n 192.168.20.40 --endpoints 192.168.20.40
support.zip already exists, overwrite? [y/N]: y
Processed with errors:
error constructing client: failed to create client connection: failed to resolve configuration context: talos config file is empty
Support bundle is written to support.zip
```
What other support generation commands can I run?

### Environment

- Talos version: v1.5.6, v1.6.6 and v1.7.0.alpha0, tried with and without bundled qemu-guest-agent
- Kubernetes version: looks like config is set to v1.29.2 but I don't think its getting that far
- Platform: Proxmox QEMU

**Comment by smira** [comment]
You haven't bootstrapped your cluster most probably, see https://www.talos.dev/v1.6/introduction/getting-started/#kubernetes-bootstrap

On talosconfig, nodes and endpoints: https://www.talos.dev/v1.6/introduction/getting-started/#understand-talosctl-endpoints-and-nodes

I would recommend reading the whole getting started document to understand the basic concepts.

**Comment by tylerobara** [comment]
Yeah, its doing more now. I will admit I jumped right to the Proxmox section.
